OpenLIFUSonicationControl/OpenLIFUSonicationControl.py

```diff
@@ -53,6 +60,57 @@ class OpenLIFUSonicationControlParameterNode:
 
     """
 
+#
+# OpenLIFUSonicationControlDialogs
+#
+
+class onRunCompletedDialog(qt.QDialog):
```

Comment:
Class names should us CamelCase starting with a captial so `OnRunCompletedDialog`